packages/datetime/test/dateRangeInputTests.tsx

```diff
@@ -1996,6 +1996,18 @@ describe("<DateRangeInput>", () => {
             assertDateRangesEqual(onChange.getCall(0).args[0], [null, null]);
             assertInputTextsEqual(root, "", "");
         });
+
+        it("Invoking openPopover opens the popover", () => {
+            const { root } = wrap(<DateRangeInput />);
+            (root.instance() as DateRangeInput).openPopover();
+            expect(root.state("isOpen")).to.be.true;
+        });
+
+        it("Invoking closePopover closes the popover", () => {
+            const { root } = wrap(<DateRangeInput />);
+            (root.instance() as DateRangeInput).closePopover();
+            expect(root.state("isOpen")).to.be.false;
+        });
```

Comment:
Mostly it's a test that the methods exist, but I also like having tests for even basic functionality if it's on the API surface area. That way no one herp-derps the correct functionality away accidentally in the future.